arangod/Cluster/ClusterMethods.cpp

```diff
@@ -2974,10 +2980,9 @@ std::vector<std::shared_ptr<LogicalCollection>> ClusterMethods::persistCollectio
     VPackBuilder velocy =
         col->toVelocyPackIgnore(ignoreKeys, LogicalDataSource::makeFlags());
 
-    infos.emplace_back(
-        ClusterCollectionCreationInfo{std::to_string(col->id()),
-                                      col->numberOfShards(), col->replicationFactor(),
-                                      waitForSyncReplication, velocy.slice()});
+    infos.emplace_back(ClusterCollectionCreationInfo{
+        std::to_string(col->id()), col->numberOfShards(), col->replicationFactor(),
+        col->minReplicationFactor(), waitForSyncReplication, velocy.slice()});
```

Comment:
You could use the variables you assigned above. Or is there somewhere a side effect such that they are no longer valid?